src/frontend/magnify/src/components/design-system/SquareAvatar/SquareAvatar.tsx

```diff
@@ -10,9 +10,28 @@ export interface SquareAvatarProps {
 
 export default function SquareAvatar({ src, title, more }: SquareAvatarProps) {
   const dataTestId = more ? 'avatar-more' : src && title ? 'avatar' : 'avatar-spacer';
+
+  // If there is no src, but a title, we want to display first letters with an accent color
+  // So we has the first letter to 1-4 and we use the corresponding color
+  const shouldFallback = !more && (!src || src === '') && title && title !== '';
+  const hash = 1 + ((title?.charCodeAt(0) || 0) % 4);
+  const background = shouldFallback ? `accent-${hash}` : undefined;
+  const initials = (title || '')
+    .split(' ')
+    .map((word) => word[0])
+    .join('');
+
   return (
-    <Avatar round="xsmall" src={src} size="40px" title={title} data-testid={dataTestId}>
+    <Avatar
+      round="xsmall"
+      src={src !== '' ? src : undefined}
+      size="40px"
+      title={title}
+      data-testid={dataTestId}
+      background={background}
+    >
       {more && <More color="brand" />}
+      {shouldFallback && initials}
```

Comment:
Just a note : those two lines lets us think that both `more` and  `shouldFallback` could be true at the same time, rendering both More and initials.
I think a good old `if` at the top of this component could be used to define a `content` variable which would be put as a child of the `Avatar` component.
Something like 
```
let content;
if(more) {
    content = (<More color="brand" />)
} else {
    content = […]
}

return (
    <Avatar […]>
        {content}
    </Avatar>
)